Sources/SimilaritySearchKit/Core/Embeddings/Models/NativeEmbeddings.swift

```diff
@@ -10,20 +10,21 @@ import NaturalLanguage
 
 @available(macOS 11.0, iOS 15.0, *)
 public class NativeEmbeddings: EmbeddingsProtocol {
-    public let model = ModelActor()
+    public let model: ModelActor
     public let tokenizer: any TokenizerProtocol
 
-    public init() {
+    public init(language: NLLanguage = .english) {
         self.tokenizer = NativeTokenizer()
+        self.model = ModelActor(language: language)
     }
 
     // MARK: - Dense Embeddings
 
     public actor ModelActor {
         private let model: NLEmbedding
 
-        init() {
-            guard let nativeModel = NLEmbedding.sentenceEmbedding(for: .english) else {
+        init(language: NLLanguage) {
```

Comment:
hope there's a way to use without specifying language

Reply:
Yes this initializer allows either specifying a language, script, or default (english)